Sources/Operators/RemoveKnownDuplicates.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  RemoveKnownDuplicates.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 21/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher where Output: Hashable {
+    /// When `Output` conforms to `Hashable`, `Publisher.removeKnownDuplicates` de-duplicates _all_
```

Comment:
```suggestion
    /// De-duplicates _all_
```

The generic constraint won't let you use this otherwise so I'd remove it from here and let folks immediately get to the beef of "what does this do"